src/katcp_fpga.py

```diff
@@ -56,7 +56,7 @@ def sendfile(filename, targethost, port, result_queue, timeout=2):
 
 class KatcpFpga(CasperFpga, async_requester.AsyncRequester, katcp.CallbackClient):
 
-    def __init__(self, host, port=7147, timeout=20.0, connect=True):
+    def __init__(self, host, port=7147, timeout=40.0, connect=True):
```

Comment:
It does no good increasing the defaulttimeout since the timeouts below specify a shorter timeout!